platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/IconButton.kt

```diff
@@ -96,6 +97,27 @@ public fun IconButton(
     )
 }
 
+@Composable
+public fun BackgroundlessIconButton(
```

Comment:
I don't think we need this tbh, users can just pass a different style to the existing composable and the rest doesn't change